txjuju/_juju1.py

```diff
@@ -48,3 +49,51 @@ def _env_as_dict(self, env):
         if env.bootstrap.admin_secret:
             config["admin-secret"] = env.bootstrap.admin_secret
         return config
+
+
+class CLIHooks(object):
+
+    CFGDIR_ENVVAR = "JUJU_HOME"
+
+    def get_bootstrap_args(self, spec, to=None, cfgfile=None,
+                           verbose=False, gui=False, autoupgrade=False):
+        # Note that for Juju 1.x we ignore gui.
+        if cfgfile is not None:
+            raise ValueError("cfgfile not supported for Juju 1.x bootstrap")
```

Comment:
I've just started looking at this, but this seems counter-intuitive to the prepare_for_bootstrap code in txjuju/__init__.py that speaks of not bothering with config files for 2.x (here, we seem to disallow them for 1.x as well, which makes me wonder when we care about them). Granted, they might be entirely different "config files", but it's not clear from looking at the diff so far if they are.